src/styles/main.css

```diff
@@ -1,3 +1,12 @@
 @tailwind base;
 @tailwind components;
 @tailwind utilities;
+
+@layer base {
+    article.blog-post h2 {
```

Comment:
`h2` elements that are children of `article` elements having a class `blog-post`.